lib/SILGen/SILGenApply.cpp

```diff
@@ -1030,6 +1030,11 @@ class SILGenApply : public Lowering::ExprVisitor<SILGenApply> {
   }
 
   void processClassMethod(DeclRefExpr *e, AbstractFunctionDecl *afd) {
+    // FIXME(backDeploy): Investigate support for back deployed class method
```

Comment:
Should be fine for final methods

Reply:
That makes sense. I'm planning to come back through with a PR specifically aimed at making sure static class methods are supported and that the attribute is rejected on methods with dynamic dispatch.